jest.config.js

```diff
@@ -43,11 +36,6 @@ module.exports = {
     // Some tests require internal files of bundled packages, which are not available
     // in production builds. They are marked using the .skip-bundled.js extension.
     ...(isPublishBundle ? ["\\.skip-bundled\\.js$"] : []),
-    ...(LIB_USE_ESM ? ["/babel-helpers/"] : []),
-    // Ignore @babel/standalone test in coverage testing because it is not built
-    ...(process.env.TEST_TYPE === "cov"
-      ? ["<rootDir>/packages/babel-standalone/"]
-      : []),
   ],
```

Comment:
```suggestion
    // Ignore @babel/standalone test in coverage testing because it is not built
    ...(process.env.TEST_TYPE === "cov"
      ? ["<rootDir>/packages/babel-standalone/"]
      : []),
  ],
```